<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Nirvana" ignored="0" name="smoke-test" tests="4" failures="1" timestamp="2022-01-18T10:07:22 EET" time="34.989" errors="0">
  <testcase name="dummyTest" time="0.11" classname="com.ali.tests.smoketest.ApiTesting">
    <failure type="java.lang.AssertionError" message="expected [3] but found [2]">
      <![CDATA[java.lang.AssertionError: expected [3] but found [2]
at com.ali.tests.smoketest.ApiTesting.dummyTest(ApiTesting.java:44)
at java.util.ArrayList.forEach(ArrayList.java:1257)
... Removed 37 stack frames]]>
    </failure>
  </testcase> <!-- dummyTest -->
  <testcase name="getTest" time="5.677" classname="com.ali.tests.smoketest.ApiTesting"/>
  <testcase name="test01" time="0.816" classname="com.ali.tests.smoketest.UiTesting"/>
  <testcase name="test02" time="10.77" classname="com.ali.tests.smoketest.UiTesting"/>
</testsuite> <!-- smoke-test -->
